src/bioregistry/data/bioregistry.json

```diff
@@ -2885,18 +2890,18 @@
     "miriam": {
       "deprecated": false,
       "description": "An Archival Resource Key (ARK) is a Uniform Resource Locator (URL) that is a multi-purpose persistent identifier for information objects of any type.",
-      "homepage": "http://n2t.net/",
+      "homepage": "https://arks.org/",
       "id": "00000592",
       "name": "ARK",
       "namespaceEmbeddedInLui": true,
       "pattern": "^(ark\\:)/*[0-9A-Za-z]+(?:/[\\w/.=*+@\\$-]*)?(?:\\?.*)?$",
       "prefix": "ark",
-      "sampleId": "/12345/fk1234",
+      "sampleId": "/53355/cl010066723",
       "uri_format": "http://n2t.net/ark:$1"
     },
     "n2t": {
       "example": "/88435/hq37vq534",
-      "homepage": "https://wiki.ucop.edu/display/Curation/ARK",
+      "homepage": "https://n2t.net/e/about.html",
```

Comment:
Since this is inside the n2t data, it will just get overwritten. Since it appears you want to have some additional context, we can use the `references` key and put this URL there.

Reply:
See ba728ad7